tests/jobs/test_scheduler_job.py

```diff
@@ -1760,7 +1760,7 @@ def evaluate_dagrun(
         dr = dag.create_dagrun(
             run_type=DagRunType.SCHEDULED,
             execution_date=dagrun_info.logical_date,
-            state=State.RUNNING,
```

Comment:
Why did you need to change these tests?

Reply:
because these tests created a scheduled dagrun in running state and this fix prevents a backfill run from overriding a scheduled DAG run.

Reply:
Ohh, yes, the scheduler tests that use `dag.run` which actually uses BackfillJob. We should fix that at some point.